### PR TITLE
docs: Deltas enabled by default

### PR DESCRIPTION
**Pull request by garethtdavies**
Change-type: patch
Signed-off-by: Gareth Davies <gareth@balena.io>

### CODE REVIEW

**Comment by garethtdavies** [comment]
@shaunmulligan a very quick update to deltas by default. Is there anything additional you would like me to cover?

File: pages/learn/deploy/delta.md [hunk 1]
```diff
@@ -6,29 +6,33 @@ thumbnail: /img/runtime/ResinSupervisorDelta.png
...
 
 Once the delta (difference between the old and new image) is calculated, the device downloads and applies this delta onto the old application image, in place. When the process is finished, the new container image (new version of the application) is started.
 
 These binary deltas save on the amount of data needed to be downloaded, reduce the storage space requirements on the device to perform an application update, and shorten the time when Docker is updating.
 
 ## Enabling delta updates
 
-Delta update behaviour is enabled with the `RESIN_SUPERVISOR_DELTA` configuration variable:
+For newly created applications, the delta update behavior is enabled by default. For existing applications, the delta update behavior is enabled with the `RESIN_SUPERVISOR_DELTA` configuration variable:
```
Comment (shaunmulligan) [inline pages/learn/deploy/delta.md]: They might need to also add `RESIN_SUPERVISOR_DELTA_VERSION` == 3, to enable the V3 version of the deltas. @dfunckt can confirm

Comment (dfunckt) [reply pages/learn/deploy/delta.md]: Users can't choose delta versions anymore @shaunmulligan -- that config var is now inaccessible by users.

File: pages/learn/deploy/delta.md [hunk 2]
```diff
@@ -6,29 +6,33 @@ thumbnail: /img/runtime/ResinSupervisorDelta.png
...
 
 Once the delta (difference between the old and new image) is calculated, the device downloads and applies this delta onto the old application image, in place. When the process is finished, the new container image (new version of the application) is started.
 
 These binary deltas save on the amount of data needed to be downloaded, reduce the storage space requirements on the device to perform an application update, and shorten the time when Docker is updating.
 
 ## Enabling delta updates
```
Comment (dfunckt) [inline pages/learn/deploy/delta.md]: I think it makes sense to flip the perspective of this document to assume the default behaviour (i.e. deltas are enabled by default for new apps) and describe how to *disable* them instead. I also think that note that's been added to the top of the document fits better here.

**Comment by dfunckt** [comment]
I think we also need to update https://www.balena.io/docs/learn/welcome/primer/#getting-to-the-devices where it reads "To reduce download time and bandwidth consumption, you can also enable delta updates."

File: pages/learn/deploy/delta.md (hunk 2 above)
Comment (garethtdavies) [reply pages/learn/deploy/delta.md]: Agreed and I've moved the note here. I've updated the title to reflect this covers enabling/disabling.  

I've left if as enabling as the focus as for me the documentation says it's enabled by default but then here's how to enable it if you don't have it... Then once everyone is at the same state i.e. it's enabled, there are instructions for disabling (which in this instance are essentially the same).  I tried this the other way and don't think it flows well, particularly as I imagine a large % do not have this enabled already.

Comment (dfunckt) [reply pages/learn/deploy/delta.md]: Sounds good, agreed.

**Comment by garethtdavies** [comment]
This is already in production right @dfunckt so this can go live?

**Comment by dfunckt** [comment]
@garethtdavies yep

**Comment by garethtdavies** [comment]
@balena-ci retest

**Pull request merged**